templates/500.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block title %}500 - Big oof{% endblock %}
+{% block content %}
+  <div id="center">
+    <h1 class="red">500 - Big oof</h1>
```

Comment:
Could we replace "Big oof" with something different?

Reply:
but it is a _big oof_, feel free to suggest something else tho

Reply:
Perhaps something *a bit* more professional, like perhaps "That doesn't seem right", based on the text you have below it?

Reply:
4ef8eb8e5d237c5c2a6f379db693de64f20410cf